### PR TITLE
[DOC] Daily Reports API

### PR DESCRIPTION
**Pull request by stevenzeiler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,19 @@ Request parameters:
 * reporter_public_key
 * ledger_hash
 
+#### Daily Reports
+Total validations, correlation coefficients
+
+#####GET /reports
+List of latest reports, one report for each validator
```
Comment (wilsonianb) [inline README.md]: Should this say past day?

Comment (stevenzeiler) [inline README.md]: how about `latest daily reports`

Comment (wilsonianb) [inline README.md]: :+1:

**Comment by wilsonianb** [comment]
:+1:

**Comment by stevenzeiler** [comment]
:+1:

**Pull request merged**